internal/engine/buildcontroller_test.go

```diff
@@ -508,13 +508,18 @@ func TestBuildControllerManualTriggerBuildReasonInit(t *testing.T) {
 	}
 }
 
-func TestBuildControllerManualTriggerWithFileChanges(t *testing.T) {
+func TestBuildControllerImageBuildTrigger(t *testing.T) {
 	for _, tc := range []struct {
-		name        string
-		triggerMode model.TriggerMode
+		name               string
+		triggerMode        model.TriggerMode
+		filesChanged       []string
```

Comment:
nit: i see no reason to specify which files if it's the same set of files every time, you might just say `filesChanged: true` 🤷‍♀  no big deal tho

Reply:
done